lib/core/src/lib/form/components/widgets/hyperlink/hyperlink.widget.html

```diff
@@ -1,7 +1,11 @@
-<div class="adf-hyperlink-widget {{field.className}}">
-    <label class="adf-label" [attr.for]="field.id">{{field.name | translate }}<span class="adf-asterisk"
-            *ngIf="isRequired()">*</span></label>
-    <div [matTooltip]="field.tooltip" matTooltipPosition="above" matTooltipShowDelay="1000">
-        <a [href]="linkUrl" target="_blank" rel="nofollow">{{linkText}}</a>
+<div class="adf-hyperlink-widget {{ field.className }}">
+    <label class="adf-label" [attr.for]="field.id">
+        {{ field.name | translate }}
+        <span class="adf-asterisk" *ngIf="isRequired()">*</span>
+    </label>
+    <div [matTooltip]="field.tooltip"
+         matTooltipPosition="above"
+         matTooltipShowDelay="{{ 1000 }}">
```

Comment:
Would be more consistent to use property binding here instead of interpolation

Reply:
👍